packages/viz/src/components/daily/reservoirtooltip/ReservoirTooltip.js

```diff
@@ -23,12 +23,15 @@ import _ from 'lodash';
 import Tooltip from '../../common/tooltips/Tooltip';
 import colors from '../../../styles/colors.css';
 import styles from './ReservoirTooltip.css';
+import { SITE_CHANGE_BY_MANUFACTURER, RESERVOIR_CHANGE, DEFAULT_MANUFACTURER } from '../../../utils/constants';
 
 class ReservoirTooltip extends React.Component {
-  renderReservoir() {
+  renderReservoir(typeOfChange) {
+    const label = (typeOfChange === RESERVOIR_CHANGE) ? 'Reservoir Change' : 'Infusion Site Change';
```

Comment:
- "Infusion Site Change" missing in translations -> "Infusion site change" instead.

Reply:
good catch
can we leverage the existing 'Infusion site changes' ? I would propose to re-use it instead of creating a new entry.

Reply:
Yes I think so, it can be updated in the translations if needed anyway.